wondrous-app/pages/organization/[username]/project.tsx

```diff
@@ -0,0 +1,50 @@
+import { useLazyQuery, useQuery } from '@apollo/client';
+import { withAuth } from 'components/Auth/withAuth';
+import EntitySidebar from 'components/Common/SidebarEntity';
+import OrgProject from 'components/organization/project';
+import { GET_ORG_FROM_USERNAME, GET_USER_PERMISSION_CONTEXT } from 'graphql/queries';
+import { useRouter } from 'next/router';
+import { useEffect, useMemo } from 'react';
+import { OrgBoardContext } from 'utils/contexts';
+
+const useGetOrgFromUsername = (username) => {
```

Comment:
i don't think this hook adds anything.
Should also use useQuery instead of useLazyQuery.
Abusing useeffect causes code to be less readable